Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -430,6 +430,219 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
             Y.Mock.verify(app);
             Y.Mock.verify(contentMock);
         },
+
+        "Should launch the universal discovery widget when receiving an moveAction event": function () {
+            this.service.on('contentDiscover', function (e) {
+                Y.Assert.isObject(e.config, "contentDiscover config should be an object");
```

Comment:
and same boolean flag issue as in https://github.com/ezsystems/PlatformUIBundle/pull/237/files#r31313336